packages/oidf-resolution-tests/__tests__/localAgent.test.ts

```diff
@@ -0,0 +1,64 @@
+import { IdentifierResolution, IIdentifierResolution } from '../../identifier-resolution/src' // FIXME fix when new types have been absorbed throughout ssi-sdk
```

Comment:
This had to be a separate package because there is no way I can add the "new JwtService()" agent plugin in package identifier-resolution